src/univariate.jl

```diff
@@ -173,6 +173,52 @@ function kde(data::RealVector; bandwidth=default_bandwidth(data), kernel=Normal,
     kde(data,dist;boundary=boundary,npoints=npoints,weights=weights)
 end
 
+function optimize(f, x_lower, x_upper; iterations=1000, rel_tol=nothing, abs_tol=nothing)
```

Comment:
Please write a brief docstring, explaining what the function does, the parameters, and what is returned; mentioning that it is not exported.